include/z64actor.h

```diff
@@ -296,33 +296,47 @@ typedef enum {
 #define ACTORCTX_FLAG_6 (1 << 6)
 #define ACTORCTX_FLAG_7 (1 << 7)
 
-
-typedef struct {
+// A Lock On entry is a set of 4 triangles which appear around an actor when the player Z-Targets it
+typedef struct TargetLockOnEntry {
```

Comment:
Does the word `Entry` need to be preserved here? I feel like the name could focus more on the triangles part? Like `LockOnTriangleSet`? or `LockOnTriangleRing`?